tests/ci/codebuild/linux-x86/pre-push.yml

```diff
@@ -6,6 +6,7 @@ version: 0.2
 phases:
   build:
     commands:
+      - ./tests/check_objects_and_errors.sh
       - go run ./tests/check_licenses.go
       - ./tests/check_generated_src.sh
       - ./tests/coding_guidelines/coding_guidelines_test.sh
```

Comment:
> Built docker image locally and verified that the script ran successfully. Checked that individual commands in the script would fail in appropriate conditions.

The Docker image is changed, were other three scripts tested to ensure they still work?